Dockerfile

```diff
@@ -1,52 +1,57 @@
 ARG PY_VERSION=3.9
 
-FROM amazon/aws-lambda-python:$PY_VERSION
+FROM amazon/aws-lambda-python:$PY_VERSION as install-stage
 
 # Declare it a second time so it's brought into this scope.
 ARG PY_VERSION=3.9
-# Get the base output file name from the command line if provided.
-ARG FILE_NAME=example
 
+# Install the Python packages necessary to install the Lambda dependencies.
+RUN python3 -m pip install --no-cache-dir \
+    pip \
+    setuptools \
+    wheel \
+  # This version of pipenv is the minimum version to allow passing arguments
+  # to pip with the --extra-pip-args option.
+  && python3 -m pip install --no-cache-dir "pipenv>=2022.9.8"
+
+WORKDIR /tmp
+
+# Copy in the dependency files.
+COPY src/py$PY_VERSION/ .
+
+# Install the Lambda dependencies.
+#
+# The --extra-pip-args option is used to pass necessary arguments to the
+# underlying pip calls.
+RUN pipenv sync --system --extra-pip-args="--no-cache-dir --target ${LAMBDA_TASK_ROOT}"
+
+FROM amazon/aws-lambda-python:$PY_VERSION as build-stage
+
+###
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
-# Note: Additional labels are added by the build workflow.
-LABEL org.opencontainers.image.authors="nicholas.mcdonnell@cisa.dhs.gov"
+###
+# github@cisa.dhs.gov is a very generic email distribution, and it is
+# unlikely that anyone on that distribution is familiar with the
+# particulars of your repository.  It is therefore *strongly*
+# suggested that you use an email address here that is specific to the
+# person or group that maintains this repository; for example:
+# LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
+LABEL org.opencontainers.image.authors="github@cisa.dhs.gov"
```

Comment:
You should update this email address, probably to the commented-out suggestion just above.

Reply:
Ah, the `first-commits` branch is still in limbo.  Nevermind.